features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/ComposeExtensionSupport.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.compose
+
+import android.view.View
+import androidx.compose.ui.platform.ComposeView
+import com.datadog.android.sessionreplay.ExtensionSupport
+import com.datadog.android.sessionreplay.SessionReplayPrivacy
+import com.datadog.android.sessionreplay.compose.mappers.ComposeWireframeMapper
+import com.datadog.android.sessionreplay.internal.recorder.OptionSelectorDetector
+import com.datadog.android.sessionreplay.internal.recorder.mapper.WireframeMapper
+
+class ComposeExtensionSupport : ExtensionSupport {
```

Comment:
why it needs to be public?

Reply:
It needs to be public to make sure customer can add it to their setup